bloom-housing/bloom
# chore: add SRO unit type

## Issue Overview

This PR addresses #1977 


## Description

This adds the SRO unit type to the database and updates various strings which previously said Single Room Occupancy.

## How Can This Be Tested/Reviewed?

This will require seeding the SRO unit type prior to testing (see included migration file). Then verify all the UI locations and CSV export mentioned in the issue.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
